SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+    %not corresponding to artificial Earth-orbiting satellites
+    %Delay public release of focal plane scientific data for an embargo period of at least 80 hours following the observation.
+    %Hold engineering and commissioning imaging data for an embargo period of at least 30 days.
+
+    \item Project team members are required to use only approved Project tools, platforms, and processes for communication, data access and analysis, documentation, software development, work management, etc.
+    In practice, we expect most work done by the Project team on the commissioning data to be done within protected directories at the Rubin US Data Facility at SLAC.
+    %Embargoed data will remain on designated project computers prior to release.
+
+  \end{itemize}
+
+  \item To maximize opportunities for public engagement, Rubin Observatory will embargo all images prior to major event releases (e.g., the System First Light celebration) as leaks severely compromise public relations effectiveness.
+
+  \item The Rubin Science Community needs timely information about the technical and scientific performance of the as-built system to prepare for their science analyses.
+
+  \item The Project team must be fully enabled to focus on time-sensitive work assignments needed to demonstrate Construction Completeness and Operational Readiness:
+
+  \begin{itemize}
+
+    \item There must exist internal communication channels to maintain free flow of information within the Project team.
+
+    \item The commissioning group will be busy with commissioning tasks and will not be able to answer broader community questions.
+
+  \end{itemize}
+
+\end{enumerate}
+
+This document provides additional details supplementing the Vera C. Rubin Observatory Data Policy \citedsp{RDO-013}, Project Publication Policy \citedsp{LPM-162} to describe appropriate information sharing practices for Rubin Observatory staff and affiliates during the on-sky commissioning period.
+% , and Rubin Observatory Data Security Standards Implementation (DMTN-199)
+
+This document also includes a proposed set of guidelines for the Rubin Observatory Project team that are intended to promote the generation and dissemination of high-quality documentation to support the LSST science community.
+
+Definitions:
+
+\begin{itemize}
+
+  \item The \textbf{Project team} includes all individuals working for Rubin Observatory who have access to proprietary engineering and on-sky commissioning data from LSSTCam prior to their release, which includes all Rubin Observatory staff as well as participants in the SIT-Com In-Kind Contribution Programs \citedsp{SITCOMTN-050}.
+
+  \item \textbf{Focal plane scientific data} corresponds to proprietary data products from on-sky imaging with LSSTCam as defined in the Rubin Data Policy \citedsp{RDO-013}.
```

Comment:
Should we also deal with comcam focal plane data?

Reply:
ComCam is included. Modified the language throughout the document to make clear that same policies apply to ComCam.